src/frame.js

```diff
@@ -0,0 +1,44 @@
+// this handles the rolls and adding them to each frame, it only knows about the one frame
```

Comment:
Great use of SRP to separate the logic for the frames and the logic for the scoring.